src/protocols/at-ps2/keyboard_interface.c

```
@@ -188,525 +188,535 @@ static enum {
- * 
+ *
  * Performance Considerations:
  * - Executes in interrupt context - minimal processing time
  * - Direct FIFO access for optimal throughput
  * - Efficient bit manipulation for frame parsing
- * 
+ *
  * @note ISR function - no blocking operations allowed
  * @note PIO FIFO automatically handles timing and bit synchronization
  */
 static void __isr keyboard_input_event_handler(void) {
```

Clang (14.0.6)

[warning] declaration uses identifier '__isr', which is a reserved identifier

(bugprone-reserved-identifier)